src/inputs-controller.ts

```diff
@@ -238,10 +271,10 @@ export class InputsController {
   /**
    * Initializes the chosen gamepad by setting its identifier in the local storage and updating the UI to reflect the chosen gamepad.
    * If a gamepad name is provided, it uses that as the chosen gamepad; otherwise, it defaults to the currently chosen gamepad.
-   * @param gamepadName - Optional parameter to specify the name of the gamepad to initialize as chosen.
+   * @param gamepadName - Name of the gamepad to initialize as chosen.
    * @param emitInitEvent - Whether to send a gamepad initialization event. Default: `true`.
    */
-  initChosenGamepad(gamepadName?: string, emitInitEvent: boolean = true): void {
+  initChosenGamepad(gamepadName: string, emitInitEvent: boolean = true): void {
     if (gamepadName) {
       this.selectedDevice[Device.GAMEPAD] = gamepadName.toLowerCase();
     }
```

Comment:
```suggestion
    this.selectedDevice[Device.GAMEPAD] = gamepadName.toLowerCase();
```
This `if` isn't necessary anymore since `gamepadName` is required, right?

Reply:
hopefully it shouldn't cause issues to remove it. Those names are provided by Phaser/the browser